events/wc_events/wc_central_kalimdor_events.txt

```diff
@@ -0,0 +1,1614 @@
+﻿namespace = tauren
+namespace = cen_kal
+
+############################################################
+#Ambushed Kodo Caravan
+tauren.5 = {
+	type = character_event
+	title = tauren_event_5_name
+	desc = tauren.5.desc
+	theme = skull
+	override_background = wilderness
+	
+	left_portrait = {
+ 		character = ROOT
+	}
+	right_portrait = {
+		character = scope:companion
+	}
+	
+	#Either in Great Barrens or Elderthalas
+	trigger = {
+		OR = {
+			title:e_great_barrens = {
+				any_in_de_jure_hierarchy = {
+					#tier = tier_county
+					holder = root
+				}
+			}
+			title:e_eldrethalas = {
+				any_in_de_jure_hierarchy = {
+					holder = root
+				}
+			}
+		}
+    }
+	
+	#Companion
+	immediate = {
+		random_knight = {
+			limit = {
+				AND = {
+					is_available_adult = yes
+					#martial >= 7
+					opinion = {
+						target = root
+						value > -40
+					}
+				}
+			}
+			save_scope_as = companion
+		}
+		
+		if = {
+			limit = {
+				NOT = { exists = scope:companion }
+			}
+			random_courtier = {
+				limit = {
+					AND = {
+						is_available_adult = yes
+						#martial >= 7
+						opinion = {
+							target = root
+							value > -40
+						}
+					}
+				}
+				save_scope_as = companion
+			}
+		}
+
+	}
+	#Investigate
+	option = {
+		name = tauren.5.a
+		
+		random_neighboring_top_liege_realm_owner = {
+			limit = {
+				AND = {
+					NOT = {
+						any_war_enemy = { this = root }
+						has_relation_rival = root
+					}
+					opinion = {
+						target = root
+						value > -30
+					}
+				}
+			}
+			save_scope_as = kodo_owner
+		}
+		
+		#If no rules acceptable opinions around
+		if = {
+			limit = {
+				NOT = { exists = scope:kodo_owner }
+			}
+			random_neighboring_top_liege_realm_owner = {
+				limit = {
+					AND = {
+						NOT = {
+							any_war_enemy = { this = root }
+							has_relation_rival = root
+						}
+					}
+				}
+				save_scope_as = kodo_owner
+			}
+		}
+		
+		
+		trigger_event = {
+			id = tauren.51
+		}
```

Comment:
use `trigger_event = { id = tauren.51 days = { 2 10 } }` for example. Recheck all your `trigger_event`s
Calling it immediately is not cool when it is chain of events